### ISSUE TITLE
do we need to lowercase the string provided by user in tnrs_match_names?

### ISSUE THREAD

**Issue by fmichonneau** [ropensci/rotl#106]
`tnrs_match_names` lowercases the string provided by the user in the results, do we need to do this?

**Comment by josephwb** [comment]
Hmm. I _think_ so? @jar398 should know. Maybe even @jimallman.

**Comment by jar398** [comment]
I don't know what `tnrs_match_names` but here is the Open Tree documentation for `match_names`, with examples.

https://github.com/OpenTreeOfLife/opentree/wiki/Open-Tree-of-Life-APIs#match_names

**Comment by jimallman** [comment]
Yes, I believe the concern is with the `search_string` field of the response (from the API example linked above):
```json
      "search_string" : "aster",
      "matched_name" : "Aster",
```
... since I'm not seeing lower-case text anywhere else. It looks like search text is always [forced to lower-case](https://github.com/OpenTreeOfLife/taxomachine/blob/d1ec2a71a22bb2629f316d5ef8af8e50d11dc7d4/src/main/java/org/opentree/tnrs/queries/MultiNameContextQuery.java#L83) by taxomachine's MultiNameContextQuery.

**Comment by josephwb** [comment]
Thanks @jimallman!
